saleor/graphql/webhook/types.py

```diff
@@ -22,6 +27,56 @@ def resolve_name(root: models.WebhookEvent, *_args, **_kwargs):
         return WebhookEventType.DISPLAY_LABELS.get(root.event_type) or root.event_type
 
 
+class EventDeliveryAttempt(CountableDjangoObjectType):
+    created_at = graphene.DateTime(required=True)
+    status = EventDeliveryStatusEnum(
+        description="Event delivery status.", required=True
+    )
+
+    class Meta:
+        description = "Webhook delivery attempts."
```

Comment:
"webhook" and "event" names are mixed in the descriptions, I would unify this and call all entities as "event deliveries" and "event delivery attempts".